.github/workflows/comments.yml

```diff
@@ -0,0 +1,27 @@
+name: comments
+
+on:
+  issue_comment:
```

Comment:
Will this job be called for every comment? 
If yes, my concern is about the [GitHub Action Usage Limits](https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration#usage-limits), As we have a total of 20 concurrent jobs. Would it be a little bit expensive?